packages/xo-server/src/xo-mixins/xen-servers.mjs

```diff
@@ -91,6 +94,13 @@ export default class {
       }
     })
 
+    this._applianceUuid = ''
+    app.hooks.on('start', async () => {
+      const value = (await XenStore.read('vm')).trim()
+      assert(value.startsWith('/vm/'))
+      this._applianceUuid = value.slice(4)
```

Comment:
I suppose you'll extend with also `applianceUrl` later on?

Reply:
I don't think so, because it's not trivial (e.g. there can be multiple IP addresses).

IMHO, by simply providing the XOA UUID, XO Lite can get all the info it needs from the XAPI.

Reply:
How's that? XOA can run anywhere else than on this XAPI. XO Lite can't guess that if it's not running here. And by definition, there's no specific reason to run your XOA locally.

Reply:
You're right, either XO Lite only displays XOAs available on the same pool or I can add more info.

Shall I add all IPs found on this machine?

Reply:
We should discuss various ideas in a quick meeting, I think that will be more efficient :)